FrontEnd/src/components/ProfileDetail/MainInfo/TitleInfo.jsx

```diff
@@ -133,15 +139,30 @@ function TitleInfo({ isAuthorized, data }) {
         </div>
         <div className={classes['title-block__company_region']}>{profile.region}</div>
       </div>
-      {isAuthorized && !ownProfile ? (
-        <button
-          onClick={handleClick}
-          type="button"
-          className={`${classes['title-block__button']} ${isSaved && classes['added_to_saved__button']}`}
-        >
-            <span className={`${classes['title-block__button--text']} ${isSaved && classes['added_to_saved__button--text']}`}>{!isSaved ? 'Додати в збережені' : 'Додано в збережені'}</span>
-          {isAuthorized ? (isSaved ? filledStar : outlinedStar) : null}
-        </button>
+      {isAuthorized ? (
+        <>
+          {!ownProfile && (
+            <button
+              onClick={handleClick}
+              type="button"
+              className={`${classes['title-block__button']} ${isSaved && classes['added_to_saved__button']}`}
```

Comment:
nit: we didn't care about accessibility so far. It might be worse to revisit our app and provide accessible attributes `tabindex`, `role`, `aria-label`. In the context of another story. But for buttons affected in this story, we might provide `tabindex`

Reply:
That's not entirely clear to me. I've tried to understand tabIndex usage and found out that there is no necessity to add tabIndex to button element unless we want to change it's default behavior. As button element has a default tabIndex value 0. 

Could you please explain, what should I do here? Should I add tabIndex attribute to the button "Додати в збережені"? What should be the value?

Reply:
The general recommendation is to use them, but since we haven't added them to any buttons so far you can skip it. We can pickup this work after MVP